Misc/NEWS.d/next/Library/2018-07-18-19-57-19.bpo-16512.Jh27PQ.rst

```diff
@@ -0,0 +1 @@
+Fix bug in imghdr.py: some jpeg files were not detecting.
```

Comment:
I am not an English native speaker, but I think this is incorrect. Let's improve this NEWS entry with something in the lines of:

> Fixed bug in ``imghdr`` that was causing some jpeg image files not being detected as such.